backend/src/test/java/de/iav/backend/service/DateTimeServiceTest.java

```diff
@@ -0,0 +1,20 @@
+package de.iav.backend.service;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class DateTimeServiceTest {
+    @Test
+    void getCurrentDateTime_whenCalled_thenReturnsCurrentDateTime() {
+        //GIVEN
+        DateTimeService dateTimeService = new DateTimeService();
+
+        //WHEN
+        java.time.LocalDateTime currentDateTime = dateTimeService.getCurrentDateTime();
```

Comment:
```suggestion
        LocalDateTime currentDateTime = dateTimeService.getCurrentDateTime();
```

und dann den import nutzen